config/hooks/pre-commit

```diff
@@ -32,12 +34,22 @@ def js_format(file_list=None):
         echo('!! Skipping JavaScript formatting because prettier is not installed.')
         return False
 
+    proc = subprocess.Popen([prettier_path, '--version'], stdout=subprocess.PIPE)
```

Comment:
In the future, might make more sense to simply use `subprocess.check_output` https://docs.python.org/2/library/subprocess.html#subprocess.check_output Which just encompasses this gross logic. It's python 2.7 specific, but we require that. :)